pkg/controllers/user/logging/deployer/appdeployer.go

```diff
@@ -83,6 +84,15 @@ func (d *AppDeployer) deploy(app *projectv3.App) error {
 		return errors.New("stale app " + app.Namespace + ":" + app.Name + " still on terminating")
 	}
 
+	app.Spec.AppRevisionName = current.Spec.AppRevisionName
+	if !reflect.DeepEqual(current.Spec, app.Spec) {
+		new := current.DeepCopy()
+		new.Spec = app.Spec
+		_, err = d.AppsGetter.Apps(app.Namespace).Update(new)
```

Comment:
Now, every time cluster sync happens we will generate the logging app spec and update the current app to it. This can override any user custom answers. We only do create before and do not update the app, so would like to confirm if this is an expected change.

Reply:
updated